src/lib/utils/thorchain/memo/addAggregatorAddressToMemo.ts

```diff
@@ -0,0 +1,60 @@
+import { MEMO_PART_DELIMITER } from './constants'
+
+type AddShortenedAggregatorToMemoArgs = {
+  /**
+   * A THORChain memo, with or without aggregator address.
+   */
+  memo: string
+  /**
+   * The aggregator address that will be shortened to two chars.
+   */
+  aggregatorAddress: string
+}
+
+// THORChain themselves use 2 characters but it might collide at some point in the future (https://gitlab.com/thorchain/thornode/-/blob/develop/x/thorchain/aggregators/dex_mainnet_current.go)
```

Comment:
suggestion: revert me in the consumptions vs. implementation - this util isn't aware of shortening

Reply:
in [2a6fc6a](https://github.com/shapeshift/web/pull/7146/commits/2a6fc6a57521f17fb2fa9ee67d3d46f222c6e53c)